apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -287,6 +335,25 @@ export const FileBrowser = React.memo(() => {
     [renameFile, project.files]
   );
 
+  const getDuplicateFilenameError = (
+    fileName: string,
+    folderId: string,
+    projectFiles: Record<string, ProjectFile>
+  ) => {
+    let message = null;
+    const existingFiles = Object.values(projectFiles).filter(
+      f => f.name === fileName && f.folderId === folderId
+    );
+    if (existingFiles.length > 0) {
+      const existingFile = existingFiles[0];
```

Comment:
There's only a single file we're checking for, right? Assuming so, this should be a find:

```
const existingFile = Object.values(projectFiles).find(
  f => f.name === fileName && f.folderId === folderId
);
if (existingFile) {
```

Reply:
good call, updating